ceno_emul/src/rv32im_encode.rs

```diff
@@ -1,116 +1,29 @@
-use crate::{InsnKind, rv32im::InsnFormat};
+use crate::{InsnKind, Instruction};
```

Comment:
maybe just merge this file with `rv32im.rs`?

Reply:
Yes, good idea.  I left it in the original position to make the PR simpler and smaller.  But we can just move this now.